external/model-preparation-algorithm/tests/ote_cli/test_classification.py

```diff
@@ -23,6 +23,7 @@
     ote_hpo_testing,
     ote_train_testing,
     ote_export_testing,
+    ote_explain_testing,
```

Comment:
This is an import but not using it. Please add it to test case.

Reply:
Thank you!